KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,47 @@
+# Kubeflow Outreach Committee (KOC)
```

Comment:
What specific powers are being delegated to the KOC from the KSC?

Or will all actions of the KOC be purely taken at the direction of the KSC, rather than making their own plans.

Reply:
All things outreach. Again, we want the KOC to feel empowered to grow the community. 

That said, they should of course be informed by the direction set by the KSC.

Reply:
Then perhaps a requirement of reporting all actions taken to the KSC (and possibly wider community) in some prescribed format is important?

Reply:
I think we can request to share the meetings notes and plans. The idea is to empower the KOC and not have them bottlenecked by KSC. Of course, the KSC should feel free to review the info at their own pace and can share their opinions when appropriate.